apps/client/src/routes/dashboards/dashboards-index/components/delete-dashboard-modal.tsx

```diff
@@ -81,132 +83,138 @@ export function DeleteDashboardModal(props: DeleteDashboardModalProps) {
   }
 
   return (
-    <Modal
-      visible={props.isVisible}
-      onDismiss={handleClose}
-      header={intl.formatMessage({
-        defaultMessage: 'Delete dashboard',
-        description: 'delete dashboard modal header',
-      })}
-      closeAriaLabel={intl.formatMessage({
-        defaultMessage: 'Close delete dialog',
-        description: 'delete dashboard modal close aria label',
-      })}
-      footer={
-        <Box float="right">
-          <SpaceBetween direction="horizontal" size="xs">
-            <Button onClick={handleClose}>
-              <FormattedMessage
-                defaultMessage="Cancel"
-                description="delete dashboard modal cancel button"
-              />
-            </Button>
+    <>
+      <Modal
+        visible={props.isVisible}
+        onDismiss={handleClose}
+        header={intl.formatMessage({
+          defaultMessage: 'Delete dashboard',
+          description: 'delete dashboard modal header',
+        })}
+        closeAriaLabel={intl.formatMessage({
+          defaultMessage: 'Close delete dialog',
+          description: 'delete dashboard modal close aria label',
+        })}
+        footer={
```

Comment:
Nit-pick: There's a lot of logic happening in multiple elements being passed in this `return` statement. Would encourage breaking this out into `const` values above and passing them in to break up the logic & improve the readability of your `return`.

Reply:
I agree. The component needs to be refactored and decomposed, as you suggest, and I will do such. It's out of scope of this change though, as only 3 lines of the component are actually changed. The diff appears large due to an increase in indentation.